float32 Xobject0
float32 Xobject1
float32 Xobject2
float32 Xobject3
float32 Xobject4
float32 Xobject5
float32 Yobject0
float32 Yobject1
float32 Yobject2
float32 Yobject3
float32 Yobject4
float32 Yobject5

